libs/dh/metering-point/feature-overview/src/lib/breadcrumbs/dh-breadcrumbs.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let transloco; read: 'meteringPoint.overview'">
+  <a [routerLink]="meteringPointAbsolutePath">{{
+    transloco('breadcrumbs.meteringPoints')
+  }}</a>
+  <watt-icon name="right"></watt-icon>
+  {{ meteringPointId }}
+</ng-container>
```

Comment:
suggestion: consider using the transloco attribute directive or pipe so that some content is shown before translatons are loaded